packages/riipen-ui-docs/src/modules/components/Menu.jsx

```diff
@@ -54,7 +55,9 @@ const Menu = () => {
           <li key={i}>
             <Accordion {...parent.props}>
               <AccordionSummary icon={Icon} iconProps={{ size: "small" }}>
-                {parent.name}
+                <Typography color="initial" fontWeight="bold" variant="h5">
```

Comment:
Make sure the parent.name doesn't randomly become white when it goes to mobile?  Some `primary` colour overrides the `inherit` colour for some reason.